### ISSUE TITLE
Tapping to open a file from a React Native stack trace opens a new VS Code window

### ISSUE THREAD

**Issue by LK** [Microsoft/vscode-react-native#757]
## Actual Behavior
1. Crash a React Native app
2. Tap a file in the resulting stack trace
3. A new VS Code window opens

## Expected Behavior
1. Crash a React Native app
2. Tap a file in the resulting stack trace
3. The file should open in the existing VS Code window

## Software versions
- React-Native VS Code extension version: 0.6.13
- VSCode version: 1.25.1
- OS platform and version: macOS 10.14 beta 4 (18A336e)
- NodeJS version: 9.11.2
- React Native version: 0.55.4

With the following settings configured, the file will open in a new window:

```
{
    "window.openFilesInNewWindow": "off",
    "window.openWithoutArgumentsInNewWindow": "off"
}
```

This is my folder setup:
```
+ workspace root
|
+ -- RN project root
|
+ -- ...
```

If I add `"window.openFoldersInNewWindow": "off"`, then the file no longer opens in a new window, but instead reloads the existing editor window, however it opens to the RN project root rather than leaving my workspace open. Manually going to a terminal and running `code /absolute/path/to/file.js` has the correct behavior of opening the file in my existing VS Code window, which makes me think this is an issue with this extension rather than VS Code itself.

**Comment by ruslan-bikkinin** [comment]
Hi @LK and thanks for submitting the issue. Could you please give us more details of your scenario so we could reproduce it on our side? So you are doing the following:
1. Open workspace that has RN project root inside.
2. Make sure you have a correct debug configuration for this project.
3. Configure your app in release mode to be able to crash it (unintentionally or not).
4. Start debugging your app using extension.
5. Get your app crashed.
6. See error log in debug console and tap a file in the resulting stack trace.
7. A new VS Code window opens, but it shouldn't though.

Is that correct?

**Comment by LK** [comment]
The app is running in debug mode, when it crashes the simulator displays the red popup window and I tap on a line of the stack trace in the simulator, which brings up a new window.

**Comment by ruslan-bikkinin** [comment]
Hi again @LK, it seems to be a VS Code issue, because I could reproduce your issue not using extension at all. In my case reproduction steps are the following:
* My demo project is called `i757`.
1. Set `REACT_EDITOR` environment variable to `code` (I did it by executing `export REACT_EDITOR=code`)
2. Start packager inside your project directory (`i757`) and run application (`react-native run-ios` or `react-native run-android` - I test it on android app)
3. Open workspace root including `i757` directory. My setup:
![image](https://user-images.githubusercontent.com/30265843/43322605-c20d481e-91b7-11e8-9c67-e429c35edc51.png)
4. When app crashes click on a line of the stacktrace.

As a result, I got this in packager output:
```
Opening /media/rbikkinin/work/support/vscode-react-native/i757/node_modules/react-native/Libraries/ReactNative/AppRegistry.js with code
```
 and then new VS Code window was opened.

So, my guess is that VS Code cannot correctly handle this situation in two conditions:
1. Command `code /path/to/file` was executed in parent process (by using `node spawn` for example).
2. Workspace multiroot that contains path to file is opened in VS Code instead of one root directory.

I believe we should address this to [vscode](https://github.com/Microsoft/vscode) folks.
